.eslintrc

```diff
@@ -1,33 +1,24 @@
 {
-  "rules": {
-    "indent": [2, 2, {
-      "VariableDeclarator": {
-        "var": 2,
-        "let": 2,
-        "const": 3
-      }
-    }],
-    "no-empty": [2],
-    "no-console": [0],
-    "comma-dangle": [2],
-    "no-unused-vars": [0],
-    "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
-  },
+  "parser": "babel-eslint",
+  "extends": "./node_modules/mofo-style/.eslintrc.yaml",
   "env": {
+    "node": true,
     "commonjs": true,
     "es6": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
   "parserOptions": {
+    "ecmaVersion": 6,
     "ecmaFeatures": {
-      "es6": true,
-      "jsx": true,
-      "experimentalObjectRestSpread": true
+      "jsx": true
     }
   },
+  "rules": {
+    "no-console": 1,
+    "react/jsx-uses-react": 1,
```

Comment:
Would get the following linting errors without using `jsx-uses-react` rule.  Any suggestion on how to fix it? :eyes: 

<img width="495" alt="screen shot 2016-02-22 at 10 13 39 am" src="https://cloud.githubusercontent.com/assets/2896608/13228967/6a59fbf4-d953-11e5-9543-4b625916797b.png">